rts/lra-examples/cdi-embedded/pom.xml

```diff
@@ -64,31 +64,22 @@
             <groupId>org.jboss.narayana.rts</groupId>
             <artifactId>narayana-lra</artifactId>
             <version>${project.version}</version>
-        </dependency>
-
 
-        <!-- MP LRA CDI support defines annotations for controlling the lifecycle
-             of LRAs and for controlling participation in an LRA
-             -->
+        </dependency>
         <dependency>
-            <groupId>org.eclipse.microprofile.lra</groupId>
-            <artifactId>microprofile-lra-api</artifactId>
-            <version>${version.microprofile.lra}</version>
+          <groupId>org.jboss.narayana.rts</groupId>
```

Comment:
This comment will duplicate something below I think but I ask here too why the MP LRA API is removed and the lra-service-base is added? If you want to answer in one place if the same answer I can review and resolve but leaving here till resolved.

Reply:
That dependency is added because it solved the previous error the cdi-participant had: java.lang.NoClassDefFoundError: io/narayana/lra/AnnotationResolver. Thanks for the review, I will update the Jira issue with this information.

Reply:
I wrote a comment about that failure here: https://github.com/jbosstm/quickstart/pull/395#issuecomment-1190305567

Reply:
May some packages from MP LRA API have been moved into lra-service-base dependency?

Reply:
I don't expect that packages would normally be moved from the Eclipse MP LRA API into Narayana dependencies so that would seem unexpected - @mmusgrov can you think some reason why we no longer need the MP LRA API? BTW this aspect should not hold up my review of this PR

Reply:
The comment about MP LRA API was:
```        <!-- MP LRA CDI support defines annotations for controlling the lifecycle
             of LRAs and for controlling participation in an LRA
             -->```

Reply:
I guess should some be clue there on why it was needed before and maybe why it is not needed now?